community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/planner/logical/steps/pickBestPlanUsingHintsAndCost.scala

```diff
@@ -20,19 +20,22 @@
 package org.neo4j.cypher.internal.compiler.v2_2.planner.logical.steps
 
 import org.neo4j.cypher.internal.compiler.v2_2.planner.logical.plans.LogicalPlan
-import org.neo4j.cypher.internal.compiler.v2_2.planner.logical.{LogicalPlanningFunction0, LogicalPlanningFunction1, CandidateSelector, LogicalPlanningContext}
+import org.neo4j.cypher.internal.compiler.v2_2.planner.logical._
```

Comment:
To give more detail, sortBy and minBy cannot be used in 2.11 anymore as our use of sorting key causes an "divergent implicit" error. Setting up our own ordering explicitly and switching to sort and min fixed that.